MAUI/Autocomplete/Migration.md

```diff
@@ -53,4 +49,52 @@ To migrate easier from Xamarin SfAutoComplete to .NET MAUI `SfAutocomplete`, we
 <td>MaximumDropDownHeight</td>
 <td>MaxDropDownHeight</td>
 <td>Gets or sets the maximum dropdown height.</td></tr>
-</table> 
\ No newline at end of file
+<tr>
+<td>SuggestionMode <br/> AutoCompleteMode</td>
+<td>TextSearchMode</td>
+<td>Gets or sets the search mode of the control. The enum values are "StartsWith" and "Contains". <br/>When using StartsWith, the AutoCompleteMode will be set to Append. <br/> When using Contains, the AutoCompleteMode will be set to Suggest.</td></tr>
+<tr>
+<td>ItemsSource,<br/>DataSource,<br/>ComboBoxSource. </td>
+<td>ItemsSource</td>
+<td>Gets or sets the items source to be populated in the control.</td></tr>
```

Comment:
ItemsSource is a single word. change as Gets or sets the data to be populated

Reply:
Changes are made